quboo.properties

```diff
@@ -20,7 +20,13 @@ defaults.mavenArtifactId=quboo-sonar-plugin
 1.0.5.changelogUrl=https://github.com/thepracticaldeveloper/quboo-sonarqube-plugin/releases/tag/v1.0.5
 
 1.0.6.description=Added two new fields to export (type, tags) and better logs to help connecting the account.
-1.0.6.sqVersions=[6.7,LATEST]
+1.0.6.sqVersions=[6.7,7.9]
 1.0.6.downloadUrl=https://github.com/thepracticaldeveloper/quboo-sonarqube-plugin/releases/download/v1.0.6/quboo-sonar-plugin-1.0.6.jar
 1.0.6.date=2019-11-01
 1.0.6.changelogUrl=https://github.com/thepracticaldeveloper/quboo-sonarqube-plugin/releases/tag/v1.0.6
+
+1.1.0.description=Added compatibility with Sonar 8
+1.1.0.sqVersions=[7.4,LATEST]
```

Comment:
This range needs to start at 7.9, the oldest version still supported in the Marketplace.

Reply:
Done